### PR TITLE
[11.0] [MIG] account move line product: Migration to 11.0

### PR DESCRIPTION
**Pull request by VRodrigo**

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
Code reivew LGTM

Please squash your migration commits into one.

Thanks!

File: account_move_line_product/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
```
Comment (HviorForgeFlow) [inline account_move_line_product/README.rst:1]: change to .png image

File: account_move_line_product/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```
Comment (HviorForgeFlow) [inline account_move_line_product/README.rst:36]: change to .png image

File: account_move_line_product/README.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+This will be relevant in perpetual inventory, for inventory-related account
+moves.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/92/10.0
```
Comment (HviorForgeFlow) [inline account_move_line_product/README.rst:20]: 10.0 -> 11.0 version change

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Review by gurneyalex — APPROVED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
Code seems fine to me, pending for runbot for functional

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by pedrobaeza** [comment]
There are still comments unattended.

**Comment by JordiBForgeFlow** [comment]
@VRodrigo can you address the minor comments? We're almost ready to merge.

**Comment by emagdalenaC2i** [comment]
@pedrobaeza I think this PR should be closed because this module is already merged in #471

**Pull request closed without merging**